### PR TITLE
docs: Rate limits, Applications, schema management and API docs

### PR DESCRIPTION
**Pull request by nataliekwong**
## What
- Adds rate limited message type
- Adds Application document
- Update schema management doc for clarity 
- Remove API doc table and center on Airbyte API

### CODE REVIEW

File: docusaurus/sidebars.js [hunk 1]
```diff
@@ -625,6 +625,7 @@ module.exports = {
       type: "category",
       label: "Integrating with Airbyte",
       items: [
+        "using-airbyte/configuring-api-access",
```
Comment (nataliekwong) [inline docusaurus/sidebars.js:628]: Should it be here, or under API documentation?

File: docs/using-airbyte/schema-change-management.md [hunk 2]
```diff
@@ -6,100 +6,120 @@ products: all
...
-| Column data type changes | The data in the destination will remain the same. For those syncing on a Destinations V2 destination, any new or updated rows with incompatible data types will result in a row error in the destination tables and show an error in the `airbyte_meta` field. You will need to refresh the schema and do a full resync to ensure the data types are consistent. |
+## Detecting Schema Changes
 
-## Detect and Propagate Schema Changes
+For Cloud users, Airbyte automatically checks for any changes in your source schema immediately before syncing. For Self-hosted users, Airbyte checks for any changes in your source schema immediately before syncing, at most every 24 hours. This means your source schema changes may not be automatically propagated if it has been less than 24 hours since the previous schema check.
```
Comment (teallarson) [inline docs/using-airbyte/schema-change-management.md:15]: Should this be `self-managed` or do we use `self-hosted` interchangeably?

File: docs/using-airbyte/schema-change-management.md [hunk 3]
```diff
@@ -6,100 +6,120 @@ products: all
...
-| Destination Format / Normalization Change | The way the destination writes the final data or how Airbyte cleans that data is changing in a way that requires a full refresh |
-| State Changes                             | The format of the source’s state has changed, and the full dataset will need to be re-synced                                    |
+1. In the Airbyte UI, click **Connections**. Select a connection and navigate to the **Schema** tab. If schema changes have been detected, you'll see a red "i" icon next to the `Schema` tab.
 
-To review and fix breaking schema changes:
+2. Click the stream to open the field details for that stream. Ensure your cursor and primary key are selected. Changes are automatically saved as soon as the fields are selected.
 
-1. In the Airbyte UI, click **Connections** and select the connection with breaking changes.
+5. Exit the field selection pane and click **Save changes** to ensure the schema changes are applied to the next sync. You will be prompted to refresh or clear the affected streams so that Airbyte can ensure future syncs are successful. Airbyte highly recommends refreshing or clearing streams and sync all historical data again.
```
Comment (teallarson) [inline docs/using-airbyte/schema-change-management.md:79]: "Changes are automatically saved..." and "click `Save changes` to ensure schema changes are applied" seems confusing.  I know what it's getting at (that the panel itself has no save button) but if all they do is step 2, nothing will _actually_ get saved.

**Review by teallarson — COMMENTED** [review]
Nice!  Couple of small comments.

File: docs/api-documentation.md [hunk 4]
```diff
@@ -4,12 +4,23 @@ products: all
...
+
+View our documentation [here](./using-airbyte/configuring-api-access.md) to learn how to start using the Airbyte API.             
+
+## Using the Airbyte API
+Navigate to our full API documentation to learn how to retrieve your access token, make API requests, and manage resources like sources, destinations, and workspaces. 
+
+Our full API documentation is located here: [api.airbyte.com](https://reference.airbyte.com/reference/getting-started).
+
+## Configuration API (Deprecated)
+The configuration API is now deprecated and no longer supported. It is an internal API that is designed for communications between different Airbyte components ratther than.
```
[the remark this answers is not in the document]
Comment (nataliekwong) [reply docs/api-documentation.md:24]: Added!

File: docs/using-airbyte/schema-change-management.md (hunk 2 above)
Comment (nataliekwong) [reply docs/using-airbyte/schema-change-management.md:15]: Fair point, editing this to self-managed

File: docs/using-airbyte/schema-change-management.md (hunk 3 above)
Comment (nataliekwong) [reply docs/using-airbyte/schema-change-management.md:79]: Agreed, I removed the auto-save here!

**Review by alex-gron — APPROVED** [review]
Thank you Natalie! ⭐

**Review by teallarson — APPROVED** [review]

**Pull request merged**